infra/gcp/bash/ensure-prod-storage.sh

```diff
@@ -87,7 +89,9 @@ readonly PROD_PROJECT_DISABLED_SERVICES=(
 )
 
 # Regions for prod GCR.
-PROD_REGIONS=(us eu asia)
+GCR_PROD_REGIONS=(us eu asia)
+# Regions for prod AR.
+AR_PROD_REGIONS=(asia-east1 asia-northeast1 europe-north1 europe-west1 europe-west4 us-central1 us-east1 us-east4)
```

Comment:
We need to add more locations.

US West Coast | us-west2 (Los Angeles)
South America | southamerica-west1 (Santiago, Chile)
Australia | australia-southeast1 (Sydney)
Southern Europe | europe-southwest1 (Madrid)
India | asia-south1 (Mumbai)

Also, is it worth preloading most regions instead of modifying https://github.com/kubernetes/k8s.io/tree/main/k8s.gcr.io/manifests everytime we need to add one?

@BenTheElder